japanese/lookup.py

```diff
@@ -5,22 +5,39 @@
 # Dictionary lookup support.
 #
 
-import urllib, re
+import re
 from anki.hooks import addHook
 from aqt import mw
 from aqt.qt import *
 from aqt.utils import showInfo
 
+# Backwards compatibility
+try:
+    from urllib import quote  # Python 2.X
+except ImportError:
+    from urllib.parse import quote  # Python 3+
+try:
+    Copy = QWebEnginePage.Copy # Python 3+
+except NameError:
+    Copy = QWebPage.Copy # Python 2.X
+try:
+    setUrl = QUrl.setEncodedUrl # Python 2.X
+except AttributeError:
+    setUrl = QUrl.setUrl # Python 3+
+
 class Lookup(object):
 
     def __init__(self):
         pass
 
     def selection(self, function):
         "Get the selected text and look it up with FUNCTION."
+        #TODO This doesn't seem to work...
```

Comment:
The action is executed asynchronously, so you need to either wait a while or grab the clipboard in javascript via evalWithCallback()